docs/pages/benchmarking_and_the_dev_process.md

```diff
@@ -0,0 +1,50 @@
+# Benchmarking and the development process
+
+Using benchmarks as a development tool is all about the context those benchmarks come from and are run in. This context is important not only to help interpret the benchmarks (did performance increase or decrease, compared to what?) they also make up the history of measurement for the benchmarks we are running.
+
+Although conbench can store benchmarks from anywhere within a git tree (or even software that is outside of a git tree), we typically benchmark at specific points within a git tree for different purposes. Further, we use the git history to situate where a benchmark came from in history, what came before it, and its relationship to various points of interest in the git tree.
+
+In the most typically conbench setup we run benchmarks at different points in a repository’s git history: 
```

Comment:
nit:
```suggestion
In the most typical conbench setup we run benchmarks at different points in a repository’s git history: 
```

Reply:
A question of style is: "we run" or "we recommend to run"?

I understand that this is tailored for usage in VD, but for a more general documentation style maybe the latter is better.